install/resources/mybb_theme.xml

```diff
@@ -323,7 +323,7 @@ a:active {
 	background-position: 0 -30px;
 }
 
-.forum_offlock {
+.forum_offclose {
```

Comment:
Note: Themes don't get upgraded by the upgrade system. We need to make a note in the release notes in **big** letters about this @mybb/developers @mybb/sqa.